code/projects/app/src/app/stages/e5/e5-services/png-meta.service.ts

```diff
@@ -0,0 +1,97 @@
+import { Injectable } from "@angular/core";
+import { Buffer } from "buffer";
+import * as crc32 from "crc-32";
+
+@Injectable({
+  providedIn: "root",
+})
+export class PngMetaService {
```

Comment:
This class scares me on many levels. Good job.

Reply:
dont care

Reply:
![dont-care-didnt-ask](https://github.com/user-attachments/assets/9ec7c29e-5ef6-4514-a906-db97ff9826ad)